10_program-part1.js

```diff
@@ -0,0 +1,17 @@
+// 10.1 Can you store multiple types in an array? Numbers and strings? Make an example that illustrates your answer.
+
+let myArray =["My age",30 , ["Student" ,5]]; // With array is possibale to store different data, within a single variable
+console.log(myArray);
+
+
+//10.2 Can you compare infinities? (Not in Eyad's world) - does 6/0 === 10/0? How can you test this?
+//10.3 Add console.log statements to the above program
+let inf1 = 10/0;
+let inf2 = 6/0;
+
+function comparInfinities(inf1,inf2){
+        if (inf2 = Number.MAX_VALUE , inf2 = Number.MAX_VALUE)
+        console.log("inf1 and inf2 are infinity")
+        else console.log("False")
+}
+comparInfinities()
```

Comment:
In the line if (inf2 = Number.MAX_VALUE , inf2 = Number.MAX_VALUE) is not doing what you think. You use an assignment operator (=) and you mean to use a comparison operator (== or ===). Also, the ',' gives a syntax error.  I think you mean the logical operator AND. Finally, I think Infinity == Number.MAX_VALUE is false. Bonus for using Number.MAX_VALUE though!